lib/bugsnag/http_client/response.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Bugsnag.HTTPClient.Response do
+  @moduledoc false
+
+  @type t :: %__MODULE__{
+          status: non_neg_integer(),
+          body: String.t(),
```

Comment:
Even though `String.t` and `binary` are the same, I'd probably use `binary` here to make it clear that it is not necessarily a UTF-8 encoded binary.